u2f-ref-code/java/tests/com/google/u2f/client/impl/U2FClientReferenceImplTest.java

```diff
@@ -81,9 +81,9 @@ public void testAuthenticate() throws Exception {
         .thenReturn(
                 new AuthenticateResponse(UserPresenceVerifier.USER_PRESENT_FLAG, COUNTER_VALUE,
                     SIGNATURE_AUTHENTICATE));
-    when(mockU2fServer.processSignResponse(
-        new SignResponse(KEY_HANDLE_BASE64, SIGN_RESPONSE_DATA_BASE64, BROWSER_DATA_SIGN_BASE64, SESSION_ID)))
-        .thenReturn(new SecurityKeyData(0L, KEY_HANDLE, USER_PUBLIC_KEY_ENROLL_HEX, VENDOR_CERTIFICATE, 0));
+    when(mockU2fServer.processSignResponse(new SignResponse(KEY_HANDLE_BASE64,
+        SIGN_RESPONSE_DATA_BASE64, BROWSER_DATA_SIGN_BASE64, SESSION_ID), 0L)).thenReturn(
+            new SecurityKeyData(0L, KEY_HANDLE, USER_PUBLIC_KEY_ENROLL_HEX, VENDOR_CERTIFICATE, 0));
```

Comment:
is '0' the time?  Would be better to define constants for it so it's easier to read.